packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.types.ts

```diff
@@ -18,6 +18,11 @@ export interface IDropdownProps extends ISelectableDroppableTextProps<HTMLDivEle
    */
   placeHolder?: string;
 
+  /**
+   * Options for the dropdown.
+   */
+  options: IDropdownOption[];

```

Comment:
Technically a breaking change since the API surface changed, but on the other hand, this is more akin to type tightening (fixing bugs in our type declaration so that they're better and catch more issues) and the way you fundamentally would expect the API surface to perform has not changed. 

Just like as in the case with the Typescript compiler, our present position is that we do not consider these as major-version-only changes.